about.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet" />
+    <title>CREATIVE CITY SUMMIT</title>
+</head>
+
+<body>
+    <header>
+        <nav class="upper-nav">
+            <ul class="upper-ul">
+                <li><a href="#"><img src="./images/facebook.png" alt="facebookIcon"></a></li>
+                <li><a href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="https://www.google.com/">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-navbar">
+            <span class="menu" id="menu"><i class="fas fa-bars"></i></span>
+            <a href="index.html"><img class="logo" src="./images/logo.png" alt="logo"></a>
+            <ul class="second-list" id="mobile-menu">
+                <li><i class="fas fa-times close-btn" id="close-btn"></i></li>
+                <li><a href="index.html" class="nav-list" id="about-link">Home</a></li>
+                <li><a href="#" class="nav-list">Program</a></li>
+                <li><a href="#" class="nav-list">Join</a></li>
+                <li><a href="#" class="nav-list">Sponsor</a></li>
+                <li><a href="#" class="button nav-list">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    </main>
```

Comment:
- [ ] [OPTIONAL] Please remove the extra closing `</main>` tag from line 37 to as doesn't have any opening tag and seems to be placed by mistake.